services/registry/builtin-resources/schemas/base.gql

```diff
@@ -24,7 +24,7 @@ directive @select(path: [String!]!) on FIELD_DEFINITION
 directive @gql(
     url: String!
     queryName: String!
-    arguments: String
+    arguments: String = ""
```

Comment:
Why is this a string? Shouldn't it be some kind of key-value mapping?

Reply:
I'll take care of this in next PR.